python/Produce_Config_File.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+from ROOT import TGraph
+from ROOT import TMultiGraph
+import ROOT as r
+import argparse
```

Comment:
Duplicate import, remove (e.g. at line 10 you import from options.py which itself imports argparse).